praetorian_cli/sdk/test/ui/test_aegis_schedules_sdk.py

```
@@ -0,0 +1,186 @@
+            end_date='2024-12-31T23:59:59Z',
+            config={'aegis': 'true', 'credential_id': 'cred-123'},
+            client_id='C.abc123',
+        )
+
+        body = api.post.call_args[0][1]
+        assert body['endDate'] == '2024-12-31T23:59:59Z'
+        assert body['config']['credential_id'] == 'cred-123'
+        assert body['clientId'] == 'C.abc123'
+
+    def test_create_returns_api_response(self):
+        sched, api = self._make_schedules()
```

Ruff (0.14.14)

[warning] Unpacked variable `api` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)